src/Jobs/JobDefinitions/JobDefinition.php

```diff
@@ -8,40 +8,19 @@
 
 class JobDefinition implements JobDefinitionInterface
 {
-    /**
-     * @var string
-     */
-    private $jobClass;
-
-    /**
-     * @var string
-     */
-    private $queueName;
-
-    /**
-     * @var int|null
-     */
-    private $maxAttempts;
-
-    /**
-     * @var JobLoaderInterface
-     */
-    private $jobLoader;
-
-    /**
-     * @var DelayRuleInterface
-     */
-    private $delayRule;
-
-    /**
-     * @var JobProcessorInterface
-     */
-    private $jobProcessor;
-
-    /**
-     * @var string
-     */
-    private $jobName;
+    private string $jobClass;
+
+    private string $queueName;
+
+    private ?int $maxAttempts = null;
```

Comment:
[o] Not necessary to initialize during the property declaration (it's always initialized in the constuctor).
```suggestion
    private ?int $maxAttempts;
```